ArduCopter/mode.cpp

```diff
@@ -493,10 +493,12 @@ void Mode::zero_throttle_and_hold_attitude()
     attitude_control->set_throttle_out(0.0f, false, copter.g.throttle_filt);
 }
 
-void Mode::make_safe_spool_down()
+void Mode::make_safe_ground_handling(bool flight_idle)
```

Comment:
regarding naming, I think we should rename the argument to "bool spool_down" and then the true/false is swapped.  So for most calls it should be "true".  For the few calls that are changed the argument would be !copter.is_tradheli()